opm/input/eclipse/EclipseState/Grid/TransMult.cpp

```diff
@@ -164,6 +167,25 @@ namespace Opm {
         m_multregtScanner.applyNumericalAquifer(aquifer_cells);
     }
 
+data::Solution TransMult::convertToSimProps(std::size_t grid_size,
+                                            bool include_all_multminus) const {
+        data::Solution solution{false}; // not in si to prevent conversions
```

Comment:
Just for the record, solution arrays registered with `measure::identity` never undergo unit conversion.

Reply:
thanks.